lib/zendesk_api/actions.rb

```diff
@@ -276,15 +276,19 @@ def update!(client, attributes = {}, &block)
   module UpdateMany
     # Updates multiple resources using the update_many endpoint.
     # @param [Client] client The {Client} object to be used
-    # @param [Array] ids An array of ids to update
+    # @param [Array] ids_or_attributes_array An array of ids or arributes including ids to update
     # @param [Hash] attributes The attributes to update resources with
     # @return [JobStatus] the {JobStatus} instance for this destroy job
-    def update_many!(client, ids, attributes)
+    def update_many!(client, ids_or_attributes_array, attributes = {})
```

Comment:
`ids_or_attributes_array` naming looks not good to me...

Reply:
I'd leave off the _array ... and make `attributes=nil` to mke the check simpler

Reply:
Sounds good to me but L283 might need conditions.